### PR TITLE
Issue template for NOS

### PR DESCRIPTION
**Pull request by Mecharyry**
## This PR adds
An issue template.

## Considerations and implementation
Opening this mainly for discussion on the important pieces of information that we typically require when someone opens an issue against our open source projects. The plan is for this to be used as a starting point for all NOS projects.  

# Screen Capture

screen shot 2017-12-01 at 13 07 20

### Paired with 
Nobody.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## Problem
+_Briefly describe the issue you are experiencing / feature you would like to see added. Although the following sections are **Optional** the more information you give, the faster we can resolve this issue for you_
```
Comment (takecare) [inline .github/ISSUE_TEMPLATE.md]: i'd change the last bit to something like `(...) the more information you provide, the faster this issue can be addressed.` as everybody is welcome and encouraged to participate/help (i.e. even if they're not part of novoda)

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+## Environment [Optional]
+* _Version of the project where the issue occurs_
+* _Real device or emulator_
+* _Mobile platform/Version_
+
+## Potential Solution [Optional]
+_Explain high level any potential solutions to the issue as you see it_
+
+## Code to Reproduce the Issue [Optional]
+_Sample code can be given to aid us understand all issues. In particular, sample code allows us to reproduce the issue making it that much easier and faster for us to resolve the issue._
```
Comment (takecare) [inline .github/ISSUE_TEMPLATE.md]: `In particular, sample code facilitates reproducing the issue, making it that much easier and faster for the issue to be addressed.`

**Comment by Mecharyry** [comment]
Thanks for responding @takecare, I've reworded based on your comments.

**Comment by takecare** [comment]
@Mecharyry can i merge or should we wait for more input?

**Comment by Mecharyry** [comment]
@takecare I'd like some more input as this will be used as a base for all our NOS projects.

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+## Problem
```
Comment (ataulm) [inline .github/ISSUE_TEMPLATE.md]: Does problem/solution imply that it's only bugs? I had a question that was opened on espresso-support last week.

Comment (Mecharyry) [reply .github/ISSUE_TEMPLATE.md]: What would you suggest? 👅

**Comment by blundell** [comment]
my _issue_ with a lot of issues opened, is that they cannot be reproduced, StackOverflow have an explanation of this down to a T:

https://stackoverflow.com/help/mcve

can we link it / or take some ideas from it?

**Comment by Mecharyry** [comment]
Good thinking @blundell we could add some of that to the `code to reproduce` section 😃

**Comment by Mecharyry** [comment]
@blundell I've updated the template based on your recommendations, let me know if you think it could be improved. 😃

**Comment by Mecharyry** [comment]
No more comments. Merging.

**Pull request merged**